synapse/daemon.py

```diff
@@ -429,24 +431,47 @@ def _loadSessConf(self, info):
 
             self.onfini( core.fini )
 
+    def _onTelePushFiniMesg(self, sock, mesg):
+        name = mesg[1].get('name')
+        guid = mesg[1].get('guid')
+        current_guid = self.pushed_guids.get(name, None)
+
+        # user = sock.get('syn:user')
+        # if not self._isUserAllowed(user, 'tele:push:fini:'+name):
+        #     return
+        #     # XXX Not certain what sould be expected here since we're
```

Comment:
We should probably record the user on the push and require the same ( even if that's None ) on the fini